composer.json

```diff
@@ -53,7 +53,7 @@
     "require-dev": {
         "phpunit/phpunit": "~4.7",
         "phploc/phploc": "~2.1",
-        "squizlabs/php_codesniffer": "~2.3"
+        "squizlabs/php_codesniffer": "<=2.3"
```

Comment:
When I ran composer update there was a PHPCS error about a missing file doc comment in a file that had a file doc comment. I didn't care to figure out why so I did that.